spec/unit/pushprocessor.spec.ts

```diff
@@ -287,6 +287,7 @@ describe("NotificationService", function () {
         expect(actions.tweaks.highlight).toEqual(true);
     });
 
+    // TODO: This is not spec compliant behaviour.
```

Comment:
Please can we have a link to the relevant part of the spec and a sentence on what we do wrong here?